src/transaction.cpp

```diff
@@ -495,7 +495,7 @@ int wsrep::transaction::before_commit()
 
         if (ret == 0 && state() == s_prepared)
         {
-            ret = certify_commit(lock);
+            ret = certify_commit(lock, nullptr);
```

Comment:
I see that the logic is not changed here, but
1. how can we be sure that certify_commit() is not called here for the second time after before_prepare() above?
2. why nullptr this time?